helm/values.yaml

```diff
@@ -106,6 +102,20 @@ entityServiceConfig:
     host: attribute-service
     port: 9012
 
+database:
+  type: mongo
+  mongo:
+    authEnabled: true
+    endpoints:
+      - host: mongo
+        port: 27017
+    replicaSet: rs0
```

Comment:
@suresh-prakash Why do we need to set the replica set here?

Reply:
This is no longer required to be a part of the service. This has been moved to cluster-level configuration. Please refer to the continuation PR.